From 0d115453cf45bf1ec87ae1fd0c2b3d44d493b649 Mon Sep 17 00:00:00 2001
From: Urvishkumar Pandya <quic_urvishku@quicinc.com>
Date: Tue, 16 Jul 2024 14:26:42 +0530
Subject: [PATCH] feeds.conf: update feed packages

Enable afc, odd, and rsrcmgr package for hazelnut branch
OpenWrt 24.0x

Change-Id: I74eb0d4f654cc85b113dd2ac1a3b9f0c9ee6bdf5
Signed-off-by: Urvishkumar Pandya <quic_urvishku@quicinc.com>
---
 feeds.conf | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/feeds.conf b/feeds.conf
index f75fb1aa479f..2a2d6bff7676 100644
--- a/feeds.conf
+++ b/feeds.conf
@@ -27,7 +27,7 @@ src-link wlan_utils ../qca/feeds/wlan_utils
 src-link sigma_dut_open ../qca/feeds/sigma-dut-open
 src-link control_app_open ../qca/feeds/control-app-open
 #src-link bluetopia ../qca/feeds/bluetopia
-#src-link odd ../qca/feeds/odd
+src-link odd ../qca/feeds/odd
 src-link qca_lib ../qca/feeds/qca-lib
 src-link snpe ../qca/feeds/snpe
 #src-link ieee1905_security ../qca/feeds/ieee1905-security
@@ -51,8 +51,8 @@ src-link nss_dpdk ../qca/feeds/nss-dpdk
 #src-link optic_drivers_prop ../qca/feeds/optic-drivers-prop
 #src-link em_ext_ctrl ../qca/feeds/em-ext-ctrl
 #src-link qca_son_mem_debug ../qca/feeds/qca-son-mem-debug
-#src-link afc ../qca/feeds/afc
-#src-link qca_rsrcmgr ../qca/feeds/qca-rsrcmgr
+src-link afc ../qca/feeds/afc
+src-link qca_rsrcmgr ../qca/feeds/qca-rsrcmgr
 #src-link qca_afc_daemon ../qca/feeds/qca-afc-daemon
 #src-link lte_data_oss ../qca/feeds/lte-data-oss
 #src-link lte_data_prop ../qca/feeds/lte-data-prop
-- 
2.34.1

